source/includes/fundamentals/code-snippets/vectorSearchQuery.go

```diff
@@ -0,0 +1,85 @@
+
+	// Converts the BSON vector to a BSON binary vector
+	queryVector := vectorFloat.Binary()
+	// end-binary-vector
+
+	// start-convert-back-vector
+	// Converts the BSON binary vector back to a BSON vector
+	backToVectorFloat, err := bson.NewVectorFromBinary(queryVector)
```

Comment:
The unused variable, `backToVectorFloat`, will cause a compiling failure. However, I'm not sure how to demonstrate the conversion while keeping the code compilable. How do you like to print the value of `backToVectorFloat.Type()`?

Reply:
Yes can add a print line!